pkg/crc/machine/machine.go

```diff
@@ -111,24 +111,23 @@ func (client *client) Start(startConfig StartConfig) (StartResult, error) {
 	}
 
 	// Pre-VM start
-	var pullSecret string
 	exists, err := client.Exists(startConfig.Name)
 	if err != nil {
 		return startError(startConfig.Name, "Cannot determine if VM exists", err)
 	}
 	if !exists {
+		_, err = startConfig.PullSecret.Value()
```

Comment:
This deserves a comment such as `// Ask early for pull secret if it hasn't been requested yet`, as it's not obvious that a call to `Value()` could display a prompt

Reply:
done :+1: